ama-sample.conf

```diff
@@ -1,17 +1,24 @@
 {
+  #required
   pg => 'postgresql://mojo:mojo@/test1',
+  #recommended
   secrets => ['mojo'],
+  #leave blank to remove login functionality
   oauth2 => {
```

Comment:
It might be good to provide a link or instructions to how to obtain a key / secret for these providers.